### DISCUSSION TITLE
Fader shape "logarithmic fast start"

### DISCUSSION THREAD

**Discussion by RM-FM** [savonet/liquidsoap#3690] [Ideas]
Hi,

supported fader shapes according to the documentations are linear, sinusoidal, logarithmic or exponential. Logarithmic fade out with fast initial volume decrease that gradually slows, is known as the most natural-sounding one. The existing logarithmic and exponential seem to do the opposite (slow initial decrease then speeding up). Any change to enhance the fader shapes with the "logarithmic fast start" variant?

**Comment by toots** [comment]
Yeah that was actually pretty straight forward to add:
* `main`: https://github.com/savonet/liquidsoap/pull/3691
* `rolling-release-v2.2.x`: https://github.com/savonet/liquidsoap/pull/3692
